project/src/components/card/card.tsx

```diff
@@ -26,12 +28,12 @@ function Card(): JSX.Element {
         </div>
         <div className="place-card__rating rating">
           <div className="place-card__stars rating__stars">
-            <span style={{width: '80%'}}></span>
+            <span style={{ width: '80%' }}></span>
             <span className="visually-hidden">Rating</span>
           </div>
         </div>
         <h2 className="place-card__name">
-          <a href="#">Beautiful &amp; luxurious apartment at great location</a>
+          <Link to={ AppRoute.Room }>Beautiful &amp; luxurious apartment at great location</Link>
```

Comment:
У пропсов обычно не ставят пробел + выше у тебя без пробела было
```suggestion
          <Link to={AppRoute.Room}>Beautiful &amp; luxurious apartment at great location</Link>
```